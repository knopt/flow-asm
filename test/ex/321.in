39
1
-2.4841786679147653
-0.16199039869190024 0.761447584578363 -3.7299368499909846 -8.904925557596142 5.51419488558852 2.1594299969380177 -5.3529346735119265 -3.3328950786542944 -4.388007348423089 -4.621383536786407 -6.195081923266699 -1.7749294119684205 5.760578831535124 -7.003705004660851 6.630310870100175 3.1614659742130833 8.842406241091801 5.565696994836088 -8.566992824673857 2.0744587528879705 5.247142341454111 4.173536034293138 1.9501449443055208 7.32869614375106 -4.044568568063555 7.381695857754721 5.88744866853985 -3.343164634326441 1.5638792711090073 6.11672923856387 -2.8232476576061583 -4.844943304300435 -1.6449506978338668 -4.955960466527718 8.082376687953332 7.360940089631427 6.021185167872517 -7.2000464968488505 -2.96673657048077 

1.1739256518995393 
