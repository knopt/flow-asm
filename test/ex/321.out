-3.480644553696097 3.055432523760583 -14.887338251752578 -21.760522111728676 41.333866300641546 -6.174405375317011 -24.014990733440214 1.6852441912342204 -7.009094741237856 -5.201131685517821 -10.104429884692925 9.20551916552864 24.480127661942493 -38.712466621748135 40.49964226419279 -5.455764518454645 22.954876865769926 -2.5742342156714244 -43.67511939455663 28.50972575751499 13.128655232213385 1.5065061483050992 -3.573155771873475 20.689978297701042 -32.29779014970119 35.76657819832584 2.1754716765006696 -26.27365729307714 13.753853063512906 17.426822005931676 -25.0317475547226 -9.867196502834505 6.304402672635351 -13.181100303174382 40.47193571219566 5.568762681827158 2.6929945710052166 -40.04394816190889 7.54956164327464 
